spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -95,363 +116,368 @@
-            and_return(result_that_failed_instant_verify)
-          allow(instance).to receive(:user_can_pass_after_state_id_check?).
-            with(result_that_failed_instant_verify).
-            and_return(true)
-          allow(result_that_failed_instant_verify).to receive(:success?).
-            and_return(false)
+          allow(instance).to receive(:with_state_id_address).and_return(transformed_pii)
+          allow(instance).to receive(:state_id_proofer).and_return(aamva_proofer)
+          allow(instance).to receive(:resolution_proofer).and_return(instant_verify_proofer)
+          allow(instant_verify_proofer).to receive(:proof).
+            and_return(residential_instant_verify_proof)
+          allow(residential_instant_verify_proof).to receive(:success?).and_return(true)
         end
 
-        context 'the failure can be covered by AAMVA' do
-          before do
-            allow(result_that_failed_instant_verify).
-              to receive(:attributes_requiring_additional_verification).
-              and_return([:address])
-          end
+        it 'only makes one request to LexisNexis InstantVerify' do
+          expect(instant_verify_proofer).to receive(:proof).exactly(:once)
+          expect(aamva_proofer).to receive(:proof)
 
-          context 'it is not covered by AAMVA' do
-            let(:failed_aamva_proof) { instance_double(Proofing::StateIdResult) }
-            before do
-              allow(aamva_proofer).to receive(:proof).and_return(failed_aamva_proof)
-              allow(failed_aamva_proof).to receive(:verified_attributes).and_return([])
-              allow(failed_aamva_proof).to receive(:success?).and_return(false)
-            end
-            it 'indicates the aamva check did not pass' do
-              result = subject
+          subject
+        end
 
-              expect(result.state_id_result.success?).to eq(false)
-            end
-          end
+        it 'produces a result adjudicator with correct information' do
+          expect(aamva_proofer).to receive(:proof)
 
-          context 'it is covered by AAMVA' do
-            let(:successful_aamva_proof) { instance_double(Proofing::StateIdResult) }
-            before do
-              allow(aamva_proofer).to receive(:proof).and_return(successful_aamva_proof)
-              allow(successful_aamva_proof).to receive(:verified_attributes).
-                and_return([:address])
-              allow(successful_aamva_proof).to receive(:success?).and_return(true)
-            end
-            it 'indicates aamva did pass' do
-              result = subject
+          result = subject
+          expect(result.same_address_as_id).to eq('true')
+          expect(result.ipp_enrollment_in_progress).to eq(true)
+          expect(result.resolution_result).to eq(result.residential_resolution_result)
+        end
 
-              expect(result.state_id_result.success?).to eq(true)
-            end
-          end
+        it 'transforms PII correctly' do
```

Comment:
New expectation checking for transformation of PII.

Reply:
i think we should add a spec to the remote context that ensures pii is not transformed